### PR TITLE
feat(dotnet-sdk): add .NET SDK package

### PR DESCRIPTION
**Pull request by tecton**
Add .NET SDK package for VS Blazor project.

### CODE REVIEW

**Review by zyxiaoyuer — APPROVED** [review]

File: packages/dotnet-sdk/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+Welcome and thank you for your interest in contributing to **Microsoft.TeamsFx.Blazor**! Before contributing to this project, please review this document for policies and procedures which will ease the contribution and review process for everyone. If you have questions, please raise your issue on github.
```
Comment (therealjohn) [inline packages/dotnet-sdk/CONTRIBUTING.md:3]: Fix package name and remove "Blazor" term

File: packages/dotnet-sdk/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Built the Project
+
+Build in Visual Studio directly, or use `dotnet build` command under root folder.
+
+## How to Run Test Cases
+
+### Run test cases
+
+Right click `TeamsFxBlazor.Test` project in Visual Studio, and choose `Run Tests`.
```
Comment (therealjohn) [inline packages/dotnet-sdk/CONTRIBUTING.md:19]: Fix project name here. It no longer has "Blazor" in the name.

**Review by therealjohn — CHANGES_REQUESTED** [review]
Just a few minor nits to fix in the docs. Thanks!

File: packages/dotnet-sdk/CONTRIBUTING.md (hunk 1 above)
Comment (tecton) [reply packages/dotnet-sdk/CONTRIBUTING.md]: Thank you John! I missed this change.

**Pull request merged**